activerecord/lib/active_record/connection_adapters/sqlite3_adapter.rb

```diff
@@ -130,6 +130,10 @@ def supports_datetime_with_precision?
         true
       end
 
+      def supports_multi_insert?
+        sqlite_version >= '3.7.11'
+      end
+
```

Comment:
It looks like the test doesn't relly test that `supports_multi_insert?` returns the correct value?